packages/carbon-graphs/src/js/controls/Timeline/Timeline.js

```diff
@@ -188,7 +188,7 @@ class Timeline extends Construct {
     const containerSVG = d3
       .select(this.config.bindTo)
       .append('div')
-      .classed(styles.container, true)
+      .classed(styles.timelineContainer, true)
```

Comment:
I checked for **styles.container** usage in all timeline graph files (including test files)  and changed everywhere accordingly.

Reply:
I think I am less concerned that you accounted for all the instances in the code that needed to be updated.  My concern was more for things like reflow, or something like it which might touch multiple graphs, that might select graph containers using `styles.container` to do updates where the timeline graphs would no longer qualify for.  

I'm not sure if this situation exists but wanted to make sure we have verified that since consumers often have multiple graphs together on a view.